### ISSUE TITLE
fatal error: module 'GoogleMapsBase' not found

### ISSUE THREAD

**Issue by yokesharun** [react-community/react-native-maps#1722]
iOS build got failed due to ` fatal error: module 'GoogleMapsBase' not found` 
i followed this doc also https://github.com/airbnb/react-native-maps/blob/master/docs/installation.md
please help me to fix this issue. Thanks in advance

![simulator screen shot - iphone 6 - 2017-10-22 at 19 52 24](https://user-images.githubusercontent.com/12830078/31863804-acbf5a28-b770-11e7-89fe-a8da710f1c48.png)

**Comment by yokesharun** [comment]
These are the steps i have already done.

- cleaned the code in xocde
- took rebuild
- did `react-native link react-native-maps`
- `pod install`
- added the lines in AppDelegate.m

here is my Podfile
```

# You Podfile should look similar to this file. React Native currently does not support use_frameworks!
source 'https://github.com/CocoaPods/Specs.git'

platform :ios, '8.0'

# Change 'cruz_ios_user' to match the target in your Xcode project.
target 'cruz_ios_user' do
  rn_path = '../node_modules/react-native'

  pod 'yoga', path: "#{rn_path}/ReactCommon/yoga/Yoga.podspec"
  pod 'React', path: rn_path, subspecs: [
    'Core',
    'RCTActionSheet',
    'RCTAnimation',
    'RCTGeolocation',
    'RCTImage',
    'RCTLinkingIOS',
    'RCTNetwork',
    'RCTSettings',
    'RCTText',
    'RCTVibration',
    'RCTWebSocket',
    'BatchedBridge'
  ]

  pod 'GoogleMaps'  # Remove this line if you don't want to support GoogleMaps on iOS
  pod 'react-native-maps', path: '../node_modules/react-native-maps'
  pod 'react-native-google-maps', path: '../node_modules/react-native-maps'  # If you need GoogleMaps support on iOS
end

post_install do |installer|
  installer.pods_project.targets.each do |target|
    if target.name == 'react-native-google-maps'
      target.build_configurations.each do |config|
        config.build_settings['CLANG_ENABLE_MODULES'] = 'No'
      end
    end
    
    if target.name == "React"
      target.remove_from_project
    end
  end
end
```
these are the dependencies in package.json file

```
"dependencies": {
    "moment": "^2.19.1",
    "react": "16.0.0-beta.5",
    "react-native": "0.49.2",
    "react-native-elements": "^0.17.0",
    "react-native-maps": "^0.17.1",
    "react-native-simple-radio-button": "^2.6.2",
    "react-native-vector-icons": "^4.4.0",
    "validate.js": "^0.11.1"
  },
```

**Comment by michelalbers** [comment]
Do not link AND use cocoapods. Do one of those two :)

**Comment by rborn** [comment]
@michelalbers does it work for you ? 

Thnx :)

**Comment by michelalbers** [comment]
@rborn see #990 and this PR: https://github.com/airbnb/react-native-maps/pull/1732

**Comment by rborn** [comment]
@michelalbers @yokesharun I managed to make it work  by following this: https://codeburst.io/react-native-google-map-with-react-native-maps-572e3d3eee14 **but only** I deleted the ios, android, node_modules folder, clear watchman and run a `react-native upgrade` and `npm install` and then started the app with `yarn start --clear-cache`

**Comment by michelalbers** [comment]
@rborn which react-native version do you use?

**Comment by rborn** [comment]
@michelalbers 
~~~
"dependencies": {
    "react": "16.0.0-beta.5",
    "react-native": "0.49.3",
    "react-native-bordered-image": "^0.1.2",
    "react-native-maps": "^0.17.1",
    "react-native-render-html": "^3.5.0-rc.3",
    "react-native-router-flux": "^4.0.0-beta.22",
    "react-navigation": "^1.0.0-beta.13"
  },
~~~

**Comment by dylanitorium** [comment]
So I got stuck with this error for a few hours and got myself real wound up.

I tried all the suggestions posted I followed the guide suggested by @rborn but to the letter to no avail. 

Turns out I wasn't paying attention to a warning present by the `pod install` which was trying to tell me about conflicts in my framework search paths.

I added `$(inherited)` to the top of my Framework Search Paths for the project in xcode and this fixed my the issue for me.

Hopefully this can help someone else!
